### DISCUSSION TITLE
Diferença entre Procedure e viwer

### DISCUSSION THREAD

**Discussion by DiegoOliver94** [balta-io/forum#222] [Tira Dúvidas] [answered]
Balta, boa tarde. Tudo bem?

estou revisando algumas aulas e me gerou duas duvidas, qual a direfença entre viwer e procedure ?
e poderia dar um exemplo de quando é recomendado usar uma ao invés de outra por favor?

muito obrigado

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @DiegoOliver94, como vai?

Uma View é apenas uma visualização, enquanto uma Procedure pode executar outras ações como excluir, inserir ou atualizar dados.
